build.gradle

```diff
@@ -23,6 +23,9 @@ buildscript {
         mavenLocal()
         maven { url "https://maven.google.com"  }
         maven { url "https://github.com/ben-manes/gradle-versions-plugin/raw/mvnrepo" }
+        maven {
+            url "https://plugins.gradle.org/m2/"
+        }
```

Comment:
This repository can be removed from here. Already included in `spotbugs.gradle`

Reply:
When I remove this I get:
```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':gnd:spotbugs'.
> Unable to infer the version of SpotBugs from currently specified SpotBugs classpath: [spotbugs-gradle-plugin-3.0.0.jar]
```

I think that's because the spotbugs.gradle is imported into gnd/build.gradle and not ./build.gradle?